portfolio/src/main/java/com/google/sps/data/AUX.java

```diff
@@ -0,0 +1,97 @@
+package com.google.sps.data;
+
+import java.io.IOException;
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport;
+import com.google.api.client.googleapis.json.GoogleJsonResponseException;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.api.client.json.JsonFactory;
+import com.google.api.client.json.jackson2.JacksonFactory;
+
+import com.google.api.services.youtube.YouTube;
+import com.google.api.services.youtube.model.VideoListResponse;
+import com.google.api.services.youtube.model.SearchListResponse;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.EntityNotFoundException;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+
+import java.io.StringWriter;
+import java.io.PrintWriter;
+
+import java.security.GeneralSecurityException;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.ArrayList;
+
+import com.google.sps.data.Video;
+
+public class AUX{
+    private static String DEVELOPER_KEY;
+
+    private static final String APPLICATION_NAME = "Watch Wisely";
+    private static final JsonFactory JSON_FACTORY = JacksonFactory.getDefaultInstance();
+
+    private static void GetDevKey(){
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+
+         DEVELOPER_KEY = (String) datastore.prepare(new Query("YoutubeAPIKey"))
+            .asSingleEntity()
+            .getProperty("Key");
+        System.out.println("Dev key is: " + DEVELOPER_KEY);
+    }
+
+    public static Video VideoIdToObject(String videoId) throws IOException, GeneralSecurityException{
+        GetDevKey();
+        YouTube youtubeService = getService();
+        // Define and execute the API request
+        YouTube.Videos.List videoRequest = youtubeService.videos()
+            .list("snippet");
+        VideoListResponse videoResponse = videoRequest.setKey(DEVELOPER_KEY)
+            .setId(videoId)
+            .execute();
+            
+        String videoTitle = videoResponse.getItems().get(0).getSnippet().getTitle();
+        String videoDescription = videoResponse.getItems().get(0).getSnippet().getLocalized().getDescription();
```

Comment:
We reuse videoResponse.getItems().get(0) in these three lines, maybe make it it's own variable?